recipes/helics/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "helics" %}
+{% set version = "1.2.1" %}
+{% set checksum = "0786edb14f854d3c54da9882dcaaaec56d2370ec98abef44d211655065194e5f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/GMLC-TDC/HELICS-src/archive/v{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  skip: True  # [not osx]
```

Comment:
With this line here you are going to build this package just for ``OSX``, is that what you want?
You may try to remove that to build to every platform